.maestro/duckplayer/always_serp_carousel_duckplayer.yaml

```diff
@@ -0,0 +1,15 @@
+appId: com.duckduckgo.mobile.android
+name: "DuckPlayer: Always > SERP > Carousel > Duck Player"
+tags:
+  - duckplayer
+---
+- retry:
+    maxRetries: 0
```

Comment:
deliberately keeping `maxRetries` at 0, or do you want to bump them to 3 like all the other maestro tests?